src/board/system76/common/acpi.c

```diff
@@ -215,6 +215,7 @@ void acpi_write(uint8_t addr, uint8_t data) {
 
     case 0xBD:
         battery_set_end_threshold(data);
+        (void)battery_save_thresholds();
```

Comment:
I wasn't sure if I should make a command for saving the thresholds, to minimize the amount it could write to flash.

With this, I am expecting the values to always be changed as a pair and that the end threshold will always be changed second.

Reply:
From the code it seems that we should make the end threshold be set first since setting the start threshold has a check on not exceeding the end threshold.

Maybe we need to set these atomically together.